src/main/java/scm/address/logic/commands/FindAndExportCommand.java

```diff
@@ -115,4 +135,22 @@ private void exportData(List<Person> users, String filename) throws IOException
         }
         mapper.writeValue(Files.newOutputStream(path), data);
     }
+
+    private void exportDataAsCsv(List<Person> users, String filename) throws IOException {
+        Path path = Paths.get(filename);
+        if (Files.exists(path) && !Files.isWritable(path)) {
+            throw new IOException("File exists but is not writable: " + path);
```

Comment:
This should also be put in a constant too, I think?

Reply:
Okay! Thanks!